test/test_fx_experimental.py

```diff
@@ -1,28 +1,29 @@
+import json
+
 import torch
-from torch.fx.symbolic_trace import symbolic_trace
 from torch.fx.experimental import GraphManipulation
-from torch.fx.experimental.Partitioner import Partitioner, Device, PartitionerConfig
+from torch.fx.experimental.Partitioner import Device, Partitioner, PartitionerConfig
+from torch.fx.symbolic_trace import symbolic_trace
 from torch.testing._internal.common_utils import run_tests
 from torch.testing._internal.jit_utils import JitTestCase
 
+
 class TestFXExperimental(JitTestCase):
     def test_find_single_partition(self):
         class TestModule(torch.nn.Module):
             def forward(self, a, b):
                 return a + b
+
         m = TestModule()
         traced = symbolic_trace(m)
         a = torch.rand(1)
         b = torch.rand(1)
-        GraphManipulation.get_size_of_all_nodes(
-            traced,
-            [a, b]
-        )
+        GraphManipulation.get_size_of_all_nodes(traced, [a, b])
         partitioner = Partitioner()
         devices = [
-            Device('dev_0', 125, 0),
-            Device('dev_1', 125, 1),
-            Device('dev_2', 125, 2)
+            Device("dev_0", 125, 0),
```

Comment:
Any reason to change ' to '' ? I think I used ' everywhere, maybe I should change them to ''.

Reply:
Good catch, I think the default formatting tool in vscode prefers " over '.